package.json

```diff
@@ -2,5 +2,5 @@
   "private": true,
   "devDependencies": {
     "bulma": "^0.7.4"
-  }
+  },
```

Comment:
why add a comma here?

Reply:
It was added because I ran `yarn` I assume instead of `npm` :S